src/applications.md

```diff
@@ -5,12 +5,16 @@ Applications of Rust:
 * Game development
   * [Amethyst Game Engine](https://book.amethyst.rs/stable/)
   * [DMG-01: How to Emulate a Game Boy](https://rylev.github.io/DMG-01/public/book/)
+  * [Emergent AI](https://psichix.github.io/emergent/)
   * [Roguelike Tutorial - In Rust](https://bfnightly.bracketproductions.com/) - [Roguelike](https://en.wikipedia.org/wiki/Roguelike) game development
   * [The Specs Book](https://specs.amethyst.rs/docs/tutorials/) - an introduction to [Entity–component–system (ECS)](https://en.wikipedia.org/wiki/Entity_component_system) and the Specs API
   * [Writing NES Emulator in Rust](https://bugzmanov.github.io/nes_ebook/index.html) - NES is the Nintendo Entertainment System gaming platform
+* GPU Development
+  * [Rust GPU Dev Guide](https://embarkstudios.github.io/rust-gpu/book/)
 * GUI development
   * [GUI development with Relm4](https://aaronerhardt.github.io/relm4-book/book/)
   * [GUI development with Rust and GTK 4](https://gtk-rs.org/gtk4-rs/stable/latest/book/)
+  * [Memory Game Tutorial using the SixtyFPS GUI toolkit](https://sixtyfps.io/releases/0.1.4/docs/tutorial/rust/)
```

Comment:
I'd prefer to stick to literal titles, so we don't have to spend time thinking how to word them better:
```suggestion
  * [SixtyFPS Memory Game Tutorial (Rust)](https://sixtyfps.io/releases/0.1.4/docs/tutorial/rust/) - tutorial using the SixtyFPS GUI toolkit
```
We can always use a comment after the link if the title isn't particularly helpful. Ok?